backend/server.js

```diff
@@ -195,30 +247,46 @@ sbp('sbp/selectors/register', {
       // This is done in a queue to handle several simultaneous requests
       // reading and writing to the same key
       await appendToIndexFactory(resourcesKey)(resourceID)
+      sbp('chelonia.persistentActions/enqueue', ['backend/server/saveIndirectResourcesIndex', resourceID])
     })
   },
+  'backend/server/saveIndirectResourcesIndex': async function (resourceID: string) {
```

Comment:
There's another selector below that seems like the inverse of this, but this is named in a way that doesn't suggest it's the inverse.. would it make sense to name this as `addToIndirectResourcesIndex`?